src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countCalls = 0;
+
+  return (firstNumber) => {
+    return (secondNumber) => {
+      countCalls++;
+
+      if (countCalls <= 3 || countCalls % 2 !== 0) {
+        return firstNumber + secondNumber;
```

Comment:
Returning correct sum for odd calls should be outside of this if statement, move it to the else block.